packages/lab/src/carousel/CarouselSlide.tsx

```diff
@@ -1,76 +1,106 @@
-import { makePrefixer } from "@salt-ds/core";
+import { makePrefixer, useForkRef } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import { clsx } from "clsx";
 import {
-  type ElementType,
   type HTMLAttributes,
-  type ReactElement,
+  type ReactNode,
   forwardRef,
+  useEffect,
   useRef,
 } from "react";
+import { useCarousel } from "./CarouselContext";
+import carouselSlideCss from "./CarouselSlide.css";
 
-import { clsx } from "clsx";
+export const CarouselSlideAppearanceValues = [
```

Comment:
I think this is to match button but can we simplify this and just a union please "bordered" | "transparent"